sdk/src/types/block/protocol.rs

```diff
@@ -152,18 +214,67 @@ pub fn calc_slot_index(timestamp: u64, genesis_unix_timestamp: u32, slot_duratio
     (1 + (timestamp - genesis_unix_timestamp as u64) / slot_duration_in_seconds as u64).into()
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Packable, CopyGetters)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "camelCase")
+)]
+#[packable(unpack_error = Error)]
+#[getset(get_copy = "pub")]
+pub struct WorkScoreStructure {
+    /// Modifier for network traffic per byte.
+    data_byte: u32,
+    /// Modifier for work done to process a block.
+    block: u32,
+    /// Modifier for slashing when there are insufficient strong tips.
+    missing_parent: u32,
+    /// Modifier for loading UTXOs and performing mana calculations.
+    input: u32,
+    /// Modifier for loading and checking the context input.
+    context_input: u32,
+    /// Modifier for storing UTXOs.
+    output: u32,
+    /// Modifier for calculations using native tokens.
+    native_token: u32,
+    /// Modifier for storing staking features.
+    staking: u32,
+    /// Modifier for storing block issuer features.
+    block_issuer: u32,
+    /// Modifier for accessing the account-based ledger to transform mana to Block Issuance Credits.
+    allotment: u32,
+    /// Modifier for the block signature check.
+    signature_ed25519: u32,
+    /// The minimum count of strong parents in a basic block.
+    min_strong_parents_threshold: u32,
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Packable, CopyGetters)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "camelCase")
+)]
+#[packable(unpack_error = Error)]
+#[getset(get_copy = "pub")]
+pub struct VersionSignalingParameters {
+    window_size: u32,
```

Comment:
missing docs

Reply:
no docs on the TIP yet either

Reply:
Yeah added a comment about it https://github.com/iotaledger/tips-draft/pull/24/files#r1300615452